modules/View.js

```diff
@@ -52,10 +52,43 @@
       this.feedbackModel = new Cell();
       this.__childViews = {};
       this.__feedbackEvents = [];
+      this.__applyTimingEventListeners(['pre-dom-event-callback', 'post-dom-event-callback', 'pre-render', 'post-render']);
```

Comment:
By 'these' do you mean the the pre and post events? What are your thoughts on how to pair them up?